samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -41,6 +46,10 @@ static const char * const fp_adv_mode_description[] = {
 	[APP_FP_ADV_MODE_DISCOVERABLE] = "discoverable",
 	[APP_FP_ADV_MODE_NOT_DISCOVERABLE] = "non-discoverable",
 };
+static struct {
+	const char *trigger;
```

Comment:
Maybe we could use array of `const char *` and bitmask (index in array would be mapped to bit in bitmask). That would simplify checking if advertising is required.

Reply:
If you opt for bitmask, you could speed up the check a bit and put whole section in ROM. Still, I see that current implementation is quite simple and it's not performance critical code, so I would not insist to change it. :)